deeppavlov/models/kbqa/kb_answer_parser_wikidata.py

```diff
@@ -39,50 +34,22 @@ class KBAnswerParserWikidata(Component, Serializable):
         answer entity.
     """
 
-    def __init__(self, load_path: str, top_k_classes: int, linker: EntityLinker, classes_vocab_keys: Tuple,
-                 debug: bool = False, relations_maping_filename: str = None, templates_filename: str = None,
-                 return_confidences: bool = True, *args, **kwargs) -> None:
+    def __init__(self, top_k_classes: int, classes_vocab_keys: Tuple, debug: bool = False,
+                 *args, **kwargs) -> None:
         """
 
         Args:
-            load_path: path to folder with wikidata files
             top_k_classes: number of relations with top k probabilities
-            linker: component `deeppavlov.models.kbqa.entity_linking`
             classes_vocab_keys: list of relations predicted by `deeppavlov.models.ner.network` model
             debug: whether to print entities and relations extracted from the question
-            relations_maping_filename: file with the dictionary of ids(keys) and titles(values) of relations
-            from Wikidata
-            templates_filename: file with the dictionary of question templates(keys) and relations for these templates
-            (values)
```

Comment:
I think it's better to have these arguments passed explicitly rather than implicitly with `*args, **kwargs`